pass/disk.go

```diff
@@ -90,24 +92,33 @@ func (s *diskStore) GlobSearch(query string) ([]string, error) {
 	//	2. DOMAIN.gpg
 	//	3. DOMAIN/SUBDIRECTORY/USERNAME.gpg
 
-	matches, err := zglob.GlobFollowSymlinks(s.path + "/**/" + query + "*/**/*.gpg")
-	if err != nil {
-		return nil, err
-	}
+	items := []string{}
 
-	matches2, err := zglob.GlobFollowSymlinks(s.path + "/**/" + query + "*.gpg")
-	if err != nil {
-		return nil, err
-	}
+	for _, path := range s.paths {
+		matches, err := zglob.GlobFollowSymlinks(path + "/**/" + query + "*/**/*.gpg")
+		if err != nil {
+			return nil, err
+		}
 
-	items := append(matches, matches2...)
-	for i, path := range items {
-		item, err := filepath.Rel(s.path, path)
+		matches2, err := zglob.GlobFollowSymlinks(path + "/**/" + query + "*.gpg")
 		if err != nil {
 			return nil, err
 		}
-		items[i] = strings.TrimSuffix(item, ".gpg")
+
+		allMatches := append(matches, matches2...)
+
+		for i, match := range allMatches {
+			// TODO this does not handle identical file names in multiple s.paths
+			item, err := filepath.Rel(path, match)
```

Comment:
@erayd this is the cause of the limitation, it generates the entry name by cutting off the configured prefix. A better solution would need to account for multiple configured paths to make sure it cuts off as much as possible, but still keeping the resulting string unique among all paths.

Reply:
Mind if I open a PR against this branch with a proposed workaround?

Reply:
Please go ahead 👍

Reply:
Done - see #239.